### ISSUE TITLE
Creating Training/Test Sets for Model Accuracy Calculation

### ISSUE THREAD

**Issue by EmirhanServeren** [EmirhanServeren/capstone-customer-segmentation#1]
We need to present accuracy of the k-means model. We are going to show the model is successful or overfit/underfit. If not successful, then the model will ve revised.
